source/iopipe/json/serialize.d

```diff
@@ -692,6 +690,7 @@ T deserialize(T, JT)(ref JT tokenizer, ReleasePolicy relPol = ReleasePolicy.afte
     return result;
 }
 
+/// `Chain c` will accept a string too.
```

Comment:
Move doc comment to previous, and replace this with

/// ditto

And it will combine both together into one group.

e.g. https://dlang.org/phobos/std_range.html#.radial

Reply:
I'd also change this to:

```
Note that all arrays are also iopipes, so you can pass in a `string` for `c`.
```

And also, probably better ddoc comments for the parameters.

thinking about this, it's fine, I can fix it up later. A large doc update is needed on this anyway.